libraries/tock-tbf/src/types.rs

```diff
@@ -227,7 +227,7 @@ pub struct TbfHeaderV2Permissions<const L: usize> {
 /// A list of persistent access permissions
 #[derive(Clone, Copy, Debug)]
 pub struct TbfHeaderV2PersistentAcl<const L: usize> {
-    write_id: u32,
+    write_id: Option<core::num::NonZeroU32>,
```

Comment:
Why convert this to an `Option` though?

Reply:
Because it expresses the property that `write_id==None` means the app does not have write access. It also forces the tbf library to explicitly handle that case.